server_admin/topics/authentication/webauthn.adoc

```diff
@@ -43,6 +38,8 @@ If you want to have WebAuthn required for all users:
 
 image:images/webauthn-browser-flow-required.png[]
 
+* In the `Bindings` menu, change the browser flow to `WebAuthn Browser`
```

Comment:
The "Save" button in the previous sentence (_Select `WebAuthn Authenticator` using the dorp down and click "Save"_) should be probably rendered with background too. Now it looks like this:

![webauthn_authenticator_save](https://user-images.githubusercontent.com/8414918/72801541-fa6dcb00-3c49-11ea-9b49-d6e6141ad77a.png)

Reply:
Good point. I will fix this and few other occurences of the "Save" and will use `Save` instead, which will use the background style.